src/lib.rs

```diff
@@ -17,34 +18,51 @@ use serde::de::DeserializeOwned;
 use thiserror::Error;
 use url::Url;
 
-pub mod billing;
-pub mod invitations;
-pub mod keys;
-pub mod members;
-pub mod projects;
-mod redacted;
-mod response;
-pub mod scopes;
-pub mod transcription;
-pub mod usage;
+#[cfg(feature = "listen")]
+pub mod common;
```

Comment:
Does this not hold anything we'd use for the `manage` or `speak` features?  If it's all `listen`-only, why not put it in the `listen` module?